src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -28,73 +29,69 @@
 
 class MissingOrgMemberSerializer(Serializer):
     def serialize(self, obj, attrs, user, **kwargs):
-        return {"email": obj.email, "externalId": obj.external_id, "commitCount": obj.commit_count}
+        return {"email": obj.email, "externalId": obj.external_id, "commitCount": obj.commit__count}
```

Comment:
🤔 did yo umean to make this a double `__`?

Reply:
yes -- since i annotated the queryset using `.annotate(Count("commit"))`, the way to get the value of the annotation is with `commit__count`. i was previously setting the annotation value manually to `commit_count` by doing `.annotate(commit_count=Count("commit"))` but i think it's better to use the default naming